asset/css/markbind.css

```diff
@@ -16,3 +16,15 @@ blockquote {
         background-color: #fff;
     }
 }
+
+.radio-list-item,
+.task-list-item {
+    list-style-type: none;
+    margin: 0 0 0 -1em;
+}
```

Comment:
This block should be on line 5, based on increasing specificity (and taking into account alphabetical order).